modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/security/SecurityConfig.java

```diff
@@ -46,22 +47,26 @@
     protected void configure(HttpSecurity http) throws Exception {
         http
                 .csrf().disable()
-                .sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS)
-                .and().authorizeRequests().antMatchers("/oauth/token")
-                .permitAll().anyRequest().authenticated();
+                .sessionManagement()
+                .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
+                .and()
+                .authorizeRequests()
+                .antMatchers("/signUp").permitAll()
+                .antMatchers("/oauth/token").permitAll()
+                .anyRequest().authenticated();
```

Comment:
See comment above about using indentation in the way that the Spring Security docs do.